pages/index.html

```diff
@@ -1,12 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../stylesheets/styles.css">
+  <script src="https://kit.fontawesome.com/0dbf060281.js" crossorigin="anonymous"></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Marck+Script&family=Noto+Serif:ital,wght@0,400;0,700;1,400;1,700&family=Rubik+Distressed&family=Rubik+Microbe&family=Shadows+Into+Light+Two&display=swap"
+    rel="stylesheet">
+  <title>Vivian's Home Page</title>
 </head>
+
 <body>
-  
+  <header>
+    <p>Vivian Zhu's Lab</p>
+  </header>
+
+  <div class="nav_bar">
```

Comment:
The nav tag would be a great fit here.
```suggestion
  <nav class="nav_bar">
```